docs/tools/terraform/reference/cookbook/kafka-clickhouse-integration-recipe.rst

```diff
@@ -0,0 +1,163 @@
+Aiven for Apache Kafka® as a source for Aiven for ClickHouse®
+=============================================================
+
+This article shows by way of example how to integrate Aiven for Apache Kafka® with Aiven for ClickHouse® using `Terraform provider for Aiven <https://registry.terraform.io/providers/aiven/aiven/latest/docs>`_. An Apache Kafka® source topic is used as a data source and Aiven for ClickHouse® is used to filter or transform the raw data with a materialized view before writing it to a regular table.
+
```

Comment:
Let's give an example of the sensor data, it will be useful to understand the ``clickhouse_kafka_user_config`` terraform block below. This could be for instance:

```suggestion
.. code-block:: javascript

  {
    "sensor_id": 10000001,
    "ts": "2022-12-01T10:08:24.446369",
    "key": "cpu_usage",
    "value": 96.42
  }
```

I don't know how that renders, or if you prefer to use another language in the ``code-block`` directive.

Reply:
The suggestion should be a bit lower in the narrative, e.g. around line 10.